### PR TITLE
Profile page game grid additions

### PR DESCRIPTION
**Pull request by BenBrownCreative**
Adds game details to a users games grid when a game is clicked on. Game details data from board game geek averages and from the users plays are combined to display nicely formatted stats to the user. UI improvements for the FocusedGame component. 

image

### CODE REVIEW

File: client/src/pages/Profile/index.js [hunk 1]
```diff
@@ -37,6 +38,7 @@ const Profile = props => {
       username: USERNAME
     }
   });
+  console.log(data);
```
Comment (meyer-mcmains) [inline client/src/pages/Profile/index.js:41]:
```suggestion
```

File: client/src/components/PlayersLastPlayed/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const PlayerWrapper = styled.div`
+  display: grid;
+  grid-template-columns: 34px 1fr auto;
+`;
+
+const Score = styled.div`
+  justify-self: center;
+`;
+
+const Player = ({ name, score, win, ...props }) => {
```
Comment (meyer-mcmains) [inline client/src/components/PlayersLastPlayed/index.js:19]: You don't need to spread over props on every component, usually this is done if you are going to pass props down to another component
```suggestion
const Player = ({ name, score, win }) => {
```

File: client/src/components/GameImage/index.js [hunk 3]
```diff
@@ -74,7 +83,8 @@ GameImage.propTypes = {
   id: PropTypes.number.isRequired,
   handleSelected: PropTypes.func.isRequired,
   transitionSpeed: PropTypes.string,
-  selected: PropTypes.bool
+  selected: PropTypes.bool,
+  username: PropTypes.string
```
Comment (meyer-mcmains) [inline client/src/components/GameImage/index.js:87]: I think this should be required?
```suggestion
  username: PropTypes.string.isRequired
```

File: client/src/pages/Profile/index.js (hunk 1 above)
Comment (BenBrownCreative) [reply client/src/pages/Profile/index.js:41]: removed

File: client/src/components/PlayersLastPlayed/index.js (hunk 2 above)
Comment (BenBrownCreative) [reply client/src/components/PlayersLastPlayed/index.js:19]: good to know

**Comment by BenBrownCreative** [comment]
Pushed up a refactored GameFocused file. I added an Icons component to handle the icon and text but was having trouble getting the formatting right to send the text prop as a template string and instead opted to send the text prop as an object. I am also sending the PlayerCount component as a prop to Icons as well.

I also added components.js files to GameFocused and Icons to separate styling from the main component. Perhaps PlayerCount and PlayTime should be moved somewhere else. Could not decide on the best place for them to live.

**Comment by BenBrownCreative** [comment]
i implemented your changes and divided icons and iconWithText into separate components

File: client/src/components/PlayersLastPlayed/index.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+const PlayerWrapper = styled.div`
+  display: grid;
+  grid-template-columns: 34px 1fr auto;
+`;
+
+const Player = ({ name, score, win }) => {
+  return (
+    <PlayerWrapper>
+      <IconWrapper>{win && <Trophy />}</IconWrapper>
```
Comment (meyer-mcmains) [inline client/src/components/PlayersLastPlayed/index.js:19]: I think this should use the new Icon component you made?

File: client/src/components/GameFocused/components.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+  height: 100%;
+  display: block;
+  background-size: cover;
+  margin: 0 auto;
+  background-position: center;
+  opacity: 0.6;
+`;
+
+export const ImgWrapper = styled.div`
+  /* background: ${p => p.theme.color.darkBlueHighlight}; */
```
Comment (meyer-mcmains) [inline client/src/components/GameFocused/components.js:77]: should we remove this comment?

**Pull request merged**